helpers.go

```diff
@@ -217,11 +228,19 @@ func searchWordsByTagIds(db *sql.DB, tags string) {
 	populateTagIds(db)
 	removeEmptyTags()
 
-	if len(Tags) > 0 {
-		log.Infoln("Using tags:", Tags)
-		queryStr = fmt.Sprintf(queryStr+" left join wt as t2 on t2.word_id=t1.id WHERE t2.tag_id IN (%s) group by t1.id", TagsToString())
+	if len(tags) > 0 {
```

Comment:
Why not just check if Tags is empty? something like `if len(Tags)>0` instead?

Reply:
when searching in a blank db (with no tags inserted), Tags will have a length of 0, thus printing `No tags were given`, `tags` in the other hand has the user defined tags that we are looking for instead